### PR TITLE
chore(ci): fix small regression in #1432

### PR DESCRIPTION
**Pull request by Dattaya**
## React Boilerplate

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/mxstbr/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [x] You have followed our [**contributing guidelines**](https://github.com/mxstbr/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
- [x] double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Internal code generators and templates are updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/mxstbr/react-boilerplate/blob/master/LICENSE.md).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,11 +7,13 @@ node_js:
...
   - node ./internals/scripts/generate-templates-for-linting
-
-install:
   - npm run test
   - npm run build
 
+install:
+  - npm i -g npm@latest
+  - npm install
+
```
Comment (gihrig) [inline .travis.yml]: If we are committed to installing dependencies with yarn, what purpose does this serve?

File: .travis.yml [hunk 2]
```diff
@@ -24,6 +26,5 @@ notifications:
 after_success: 'npm run coveralls'
 
 cache:
-  yarn: true
   directories:
```
Comment (gihrig) [inline .travis.yml]: Why disable yarn cache?

**Comment by gihrig** [comment]
See comments in code...

I only slightly prefer #1455 over this PR, just because it's simpler and caches yarn.

#1432 build time:
image

#1455 build time:
image

Is that no-cache vs cache? system load, something else ???

I don't think of this is the reason for #1322 (3.4) build failure on travis...

Thoughts?

**Comment by Dattaya** [comment]
ok, now I understand what you're trying to achieve, closing it then

**Pull request closed without merging**